### PR TITLE
dts: update memory map and remove ext-uicr

### PR DESCRIPTION
**Pull request by hakonfam**
Extended UICR will not be used as its configurations will be merged
with the UICR registers in NVR.

Memory maps changes are needed to align with pre compiled
firmware.

### CODE REVIEW

File: boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_0_8_0.overlay [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nrf54h20dk_nrf54h20_0_8_0.dtsi>
```
Comment (jonathannilsen) [inline boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_0_8_0.overlay:7]: Does this work for you? For me, this resulted in a build error, whereas
```suggestion
#include "nrf54h20dk_nrf54h20_0_8_0.dtsi"
```
works

File: dts/common/nordic/nrf9280.dtsi [hunk 2]
```diff
@@ -113,14 +113,12 @@
...
 			reg = <0xfff8000 DT_SIZE_K(2)>;
 			domain = <2>;
-			ptr-ext-uicr = <&cpuapp_uicr_ext>;
 		};
 
 		cpurad_uicr: uicr@fffa000 {
 			compatible = "nordic,nrf-uicr-v2";
 			reg = <0xfffa000 DT_SIZE_K(2)>;
 			domain = <3>;
-			ptr-ext-uicr = <&cpurad_uicr_ext>;
```
Comment (anangl) [inline dts/common/nordic/nrf9280.dtsi:123]: Why aren't these nodes removed as in `nrf54h20.dtsi`?

Comment (hakonfam) [reply dts/common/nordic/nrf9280.dtsi:123]: removed them now

**Review by jonathannilsen — APPROVED** [review]

**Review by anangl — DISMISSED** [review]

**Review by carlescufi — DISMISSED** [review]

**Review by anangl — DISMISSED** [review]

**Comment by carlescufi** [comment]
Tested here: https://github.com/zephyrproject-rtos/zephyr-testing/pull/286

**Review by carlescufi — DISMISSED** [review]

**Review by fabiobaltieri — DISMISSED** [review]

**Review by 57300 — DISMISSED** [review]

**Review by carlescufi — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Pull request merged**